docs/split-testing.md

```diff
@@ -0,0 +1,189 @@
+# Setting up a split testing experiment
+
+One thing we like to do from time to time is test changes
+on specific groups of pages and see how they perform.
+We call this "split testing".
+In contrast to traditional A/B testing,
+where the change being tested is shown to a portion of the _audience_,
+in the case of split testing,
+the change is shown to everyone, but only on a subset of similar pages
+(e.g., a group of related Ask CFPB answers).
+
+
+## So you've been asked to set up a split testing experiment
+
+Good news! The infrastructure has been put in place for you to do it easily.
+It's set up so that the code being tested is isolated by a feature flag.
+
+### The flag condition
+
+In order to check whether or not a page is part of a split testing cluster,
```

Comment:
> split testing cluster

Should what this is be defined? May just as one extra sentence around [line ten](https://github.com/cfpb/cfgov-refresh/pull/4543/files#diff-767bdb4cd43ee4726bf36b549ebe30deR10):

> the change is shown to everyone, but only on a subset of similar pages
> (e.g., a group of related Ask CFPB answers). *We call this a subset of 
> pages a cluster.*

or something like.

Reply:
Updated in 3b9fcf49384c1515decb1204770ed90def95e859